desktop/appconfig.cpp

```diff
@@ -0,0 +1,55 @@
+#include "appconfig.h"
+
+#include <QLoggingCategory>
+
+
+const QStringList loggingCategories = 
+  {"UIManager", 
+    "Flexbox", 
+    "WebSocketModule", 
+    "Networking", 
+    "ViewManager", 
+    "RCTNotification", 
+    "default",
+    "RCTStatus",
+    "jsserver",
+    "status"};
+Q_LOGGING_CATEGORY(APPCONFIG, "AppConfig")
+
+AppConfig* AppConfig::appConfig = nullptr;
+
+AppConfig::AppConfig()
+  : loggingEnabled(false) {
+  
+}
+
+AppConfig* AppConfig::inst() {
+  if (appConfig == nullptr) {
+    appConfig = new AppConfig();
+  }
+
+  return appConfig;
+}
+
+bool AppConfig::getLoggingEnabled() const {
+  return loggingEnabled;
+}
+
+void AppConfig::setLoggingEnabled(bool enabled) {
+  qCDebug(APPCONFIG) << "### appconfig setLoggingEnabled " << enabled;
+  QLoggingCategory::setFilterRules(getLoggingFilterRules(enabled));
+  this->loggingEnabled = enabled;
+}
+
+QString AppConfig::getLoggingFilterRules(bool enabled) const {
+   if (enabled) {
+     return "UIManager=false\nFlexbox=false\nViewManager=false\nNetworking=false\nWebSocketModule=false";
```

Comment:
Do I understand correctly, that we intentionally won't show react-native-desktop logs in any case, even when logging is enabled?

Reply:
Yes, i think this is similar to how #7029 is implemented. Do you think it would be better to enable everything?

Reply:
I think it is totally fine, just double-checking that I get it right